src/Client/Web/Bit.Client.Web.BlazorUI.Tests/Persona/BitPersonaTests.cs

```diff
@@ -0,0 +1,167 @@
+
+            Assert.IsTrue(persona.ClassList.Contains($"bit-persona-{enabledClass}-{visualClass}"));
+        }
+
+        [DataTestMethod,
+            DataRow("Text", "SecondaryText", "TertiaryText", "OptionalText"),
+            DataRow(null, null, null, null)]
+        public void BitPersonaShouldCurrectWorkText(string text, string secondaryText, string tertiaryText, string optionalText)
+        {
+            var component = RenderComponent<BitPersonaTest>(
+                   parameters =>
+                   {
+                       parameters.Add(p => p.Text, text);
+                       parameters.Add(p => p.SecondaryText, secondaryText);
+                       parameters.Add(p => p.TertiaryText, tertiaryText);
+                       parameters.Add(p => p.OptionalText, optionalText);
+
+                   });
+            var textClassName = component.Find(".bit-Persona-primaryTex");
+            var secendryTextClassName = component.Find(".bit-Persona-secondaryText");
+            var tertiaryTextClassName = component.Find(".bit-Persona-tertiaryText");
+            var optionalTextClassName = component.Find(".bit-Persona-optionalText");
+
+            Assert.AreEqual(text, textClassName.TextContent.HasValue() ? textClassName.TextContent : null);
+            Assert.AreEqual(secondaryText, secendryTextClassName.TextContent.HasValue() ? secendryTextClassName.TextContent : null);
+            Assert.AreEqual(tertiaryText, tertiaryTextClassName.TextContent.HasValue() ? tertiaryTextClassName.TextContent : null);
+            Assert.AreEqual(optionalText, optionalTextClassName.TextContent.HasValue() ? optionalTextClassName.TextContent : null);
+        }
+
+        [DataTestMethod,
+            DataRow(BitPersonaPresenceStatus.Blocked, true),
+            DataRow(BitPersonaPresenceStatus.Blocked, false),
+            DataRow(BitPersonaPresenceStatus.Offline, true),
+            DataRow(BitPersonaPresenceStatus.Offline, false),
+            DataRow(BitPersonaPresenceStatus.Away, true),
+            DataRow(BitPersonaPresenceStatus.Away, false),
+            DataRow(BitPersonaPresenceStatus.Online, true),
+            DataRow(BitPersonaPresenceStatus.Online, false),
+            DataRow(BitPersonaPresenceStatus.Busy, true),
+            DataRow(BitPersonaPresenceStatus.Busy, false),
+            DataRow(BitPersonaPresenceStatus.DND, true),
+            DataRow(BitPersonaPresenceStatus.DND, false),]
+        public void BitPersonaPresenceStatusClassNameTest(BitPersonaPresenceStatus presenceStatus, bool isOutOfOffice)
+        {
+            var component = RenderComponent<BitPersonaTest>(
+                  parameters =>
+                  {
+                      parameters.Add(p => p.Presence, presenceStatus);
+                      parameters.Add(p => p.IsOutOfOffice, isOutOfOffice);
+                  });
+            string DetermineIcon(BitPersonaPresenceStatus presence, bool isOutofOffice)
```

Comment:
Plz, move the DetermineIcon method to out of the BitPersonaPresenceStatusClassNameTest method.